### ISSUE TITLE
Improve Coach tabs accessibility - "Reports-Activity" tabs on a Reports learner page

### ISSUE THREAD

**Issue by MisRob** [learningequality/kolibri#10255]
In #10212, we embarked on the journey toward improved accessibility of the coach plugin. Part of it is refactoring tabs logic to use new KDS tabs components that are implemented in an accessible manner. This issue is a continuation of that work.

The goal is to use the KDS tabs components for _Reports-Activity_ tabs on a learner page in the coach reports (_Coach → Reports → Learners_ → select a learner → learner page)

![report-activity](https://user-images.githubusercontent.com/13509191/225321000-d8c3e1c9-33e9-4ff9-aaf3-a0df0bb24497.png)

## Acceptance criteria

- [ ] The current implementation is refactored to use KDS tabs components (`KTabs` or the combination of `KTabsList` + `KTabsPanel`)
- [ ] The new implementation satisfies [these](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/) and [these](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-manual/) W3C guidelines (this is already provided by KDS tabs components, however in some cases, a bit of additional Kolibri logic may be needed as well - see comments below)
- [ ] The focus ring shows when navigating with a keyboard and stays on the selected tab after pressing enter

## Comments

- KDS tabs components documentation
  - [Tabs page](https://deploy-preview-426--kolibri-design-system.netlify.app/tabs/)
  - [KTabs component](https://deploy-preview-426--kolibri-design-system.netlify.app/ktabs/)
  - [KTabsList component](https://deploy-preview-426--kolibri-design-system.netlify.app/ktabslist/)
  - [KTabsPanel component](https://deploy-preview-426--kolibri-design-system.netlify.app/ktabspanel)
- You can play around with the _Lessons-Quizzes-Groups-Learners_ tabs that are already implemented with KDS components  to experience the expected behavior
![example](https://user-images.githubusercontent.com/13509191/225308774-6663d3db-680f-4211-ac98-6a550a8f94d6.png)
- In some cases, due to the routing architecture, some additional Kolibri logic may be needed to ensure seamless focus behavior. If you run into issues with the focus ring disappearing after pressing enter on a tab or similar, [useCoachTabs composable](https://github.com/learningequality/kolibri/blob/5806f4ad50db3d96fdd0030d0659861b52a63e67/kolibri/plugins/coach/assets/src/composables/useCoachTabs.js) may help

**Comment by muditchoudhary** [comment]
Hi @MisRob Can I work on this issue? or is this for only internal team? as it does not have a help-wanted label.

I played around a bit, I think I can work on it. If I can work on it? Could you please assign it to me too?

**Comment by MisRob** [comment]
Hi @muditchoudhary, yes, this would be great improvement to Kolibri's accessibility and I believe there's even more places like this.

It may be a bit more complex, but you already have some experience and there are also many references linked in the issue description that you can use as guidance (especially the original PR and useCoachTabs composable, if that's needed).

Feel free to work step by step in a draft PR and if you had any questions, we can figure out next steps there.

**Comment by MisRob** [comment]
Closed by https://github.com/learningequality/kolibri/pull/11501
